config/city-lists.json

```diff
@@ -2,7 +2,7 @@
 	{
 		"name": "Online",
 		"country": " Edition",
-		"date": "30th October 2024",
+		"date": "October 30 2024",
```

Comment:
add a comma before 2024, like in paris and london

Reply:
yeah sure! doing right away